app/containers/App.js

```diff
@@ -126,6 +127,7 @@ export class App extends Component {
   setServer = info => {
     console.log('will use', info.label, 'server');
     var serverinfo = serverdata[info.label];
+    serverinfo.environment = info.label;
```

Comment:
This will use `Development`, `Staging`, `Production`, etc. instead of `dev`, `stg`, `prd`. I'm fine with the more verbose names, just noting that the card seems to reference the shortened/lowercase versions of how we refer to the environments in most developer settings.